osc/commandline.py

```diff
@@ -561,7 +561,8 @@ def main(cls, argv=None, run=True):
         cmd.load_legacy_commands()
         if run:
             args = cmd.parse_args(args=argv)
-            cmd.run(args)
+            exit_code = cmd.run(args)
+            sys.exit(exit_code)
         else:
             args = None
         return cmd, args
```

Comment:
The `if run:` block would never reach there `return cmd, args` statement. Just wanted to point it out, not sure if that's expected.

Reply:
Thanks for checking. Yes, it is expected. `run=False` is used during tests to  check the command and arguments that would be normally executed.